tests/lib/nested.sh

```diff
@@ -1231,6 +1221,14 @@ nested_exec_as() {
     sshpass -p "$PASSWD" ssh -p "$NESTED_SSH_PORT" -o ConnectTimeout=10 -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no "$USER"@localhost "$@"
 }
 
+nested_retry() {
+    if ! nested_exec "ls ./retry" &>/dev/null; then
```

Comment:
I think this should work too:
```suggestion
    if ! nested_exec "test -e ./retry" &>/dev/null; then
```

Reply:
nice, thanks